test/e2e/publish.e2e.js

```diff
@@ -75,9 +79,45 @@ describe('Publish Commands', () => {
 		const args = ['publish', eventName, '--public'];
 		const { stdout, stderr, exitCode } = await cli.run(args);
 
-		expect(stdout).to.equal(`Published public event: ${eventName}${'\n'}`);
+		expect(stdout).to.include(`Published public event: ${eventName}${os.EOL}`);
 		expect(stderr).to.equal('');
 		expect(exitCode).to.equal(0);
 	});
+
+	it('Publishes a product event', async () => {
+		const args = ['publish', eventName, '--product', PRODUCT_01_ID];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+
+		expect(stdout).to.include(`Published private event: ${eventName} to product: ${PRODUCT_01_ID}${os.EOL}`);
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(0);
+	});
+
+	it('Publishes a private product event', async () => {
+		const args = ['publish', eventName, '--product', PRODUCT_01_ID, '--private'];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+
+		expect(stdout).to.include(`Published private event: ${eventName} to product: ${PRODUCT_01_ID}${os.EOL}`);
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(0);
+	});
+
+	it('Publishes a public product event', async () => {
+		const args = ['publish', eventName, '--product', PRODUCT_01_ID, '--public'];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+
+		expect(stdout).to.include(`Published public event: ${eventName} to product: ${PRODUCT_01_ID}${os.EOL}`);
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(0);
+	});
```

Comment:
these aren't _super_ helpful atm - i'm going to think about how we might be able to sanely incorporate actual device(s) into the mix but in the meantime, these are better than nothing 🤗